airflow/cli/commands/webserver_command.py

```diff
@@ -376,6 +376,7 @@ def webserver(args):
         )
 
         run_args = [
+            'opentelemetry-instrument',
```

Comment:
Here. In order to make this more "pluggable" we need to add a flag in the "webserver" command.

I think an "--open-telemetry" flag would be perfect. And depending on the flag specified in the webserver, command you would either add or not the "opentelemetry-instrument".

so "airflow webserver --open-telemetry" would prepend "opentelemetry-instrument".

Reply:
Here is the webserver command: https://github.com/apache/airflow/blob/main/airflow/cli/commands/webserver_command.py